From: Christoph Feck <christoph@maxiom.de>
Date: Fri, 12 Mar 2010 16:54:01 +0000
Subject: SVN_SILENT compile with Qt 4.7
X-Git-Tag: v2.2.0-beta1
X-Git-Url: http://quickgit.kde.org/?p=krusader.git&amp;a=commitdiff&amp;h=7429ec2e1a43a5d2d0c883311b181a8d3ea53dc6
---
SVN_SILENT compile with Qt 4.7

There is no QString(int) constructor
---


--- krusader/MountMan/kmountmangui.h
+++ krusader/MountMan/kmountmangui.h
@@ -112,7 +112,7 @@ private:
 /////////////////////////////////////////////////////////////////
 class fsData {
 public:
-   fsData() : Name( 0 ), Type( 0 ), MntPoint( 0 ), TotalBlks( 0 ),
+    fsData() : Name(), Type(), MntPoint(), TotalBlks( 0 ),
    FreeBlks( 0 ), Mounted( false ) {}
 
    // get information

